lib/PlaybackArea.tsx

```diff
@@ -236,7 +242,13 @@ export const PlaybackArea: React.FC<PlaybackAreaProps> = ({
   const timestamp = refresh.toString()
 
   if (format === Format.RTP_H264) {
-    const ws = wsUri(secure ? Protocol.WSS : Protocol.WS, host)
+    const ws = wsUri(
+      secure ? Protocol.WSS : Protocol.WS,
+      host,
+      searchParams(FORMAT_API[format], {
+        ...parameters,
+      }),
```

Comment:
This would also need to be optional somehow, as I guess this is specific for your setup?